plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java

```diff
@@ -956,6 +956,21 @@ public void testSettingChangeDataFeedEnabledProperty()
                 .contains("change_data_feed_enabled = true");
     }
 
+    @Test
+    public void testCreateTableWithExistingLocation()
+    {
+        String tableName = "test_legacy_create_table_" + randomNameSuffix();
+
+        assertQuerySucceeds("CREATE TABLE " + tableName + " AS SELECT 1 as a, 'INDIA' as b, true as c");
+        assertQuery("SELECT * FROM " + tableName, "VALUES (1, 'INDIA', true)");
+
+        String tableLocation = (String) computeScalar("SELECT DISTINCT regexp_replace(\"$path\", '/[^/]*$', '') FROM " + tableName);
+        assertUpdate("DROP TABLE " + tableName);
```

Comment:
`DROP TABLE` will delete the data as well. You need to use `unregister_table` procedure here.